docs-v2/package.json

```diff
@@ -44,6 +45,7 @@
     "build:all": "npm run build:css && npm run build:js",
     "preprocess-schemas": "shx rm -rf tmp/raml* && shx mkdir -p tmp/raml && shx cp -rf ../src/main/resources/raml/* tmp/raml && shx rm -rf tmp/raml/schemas/* && node bin/bundle-schemas -i ../src/main/resources/raml/schemas -o tmp/raml/schemas",
     "raml2html": "npm run preprocess-schemas && raml2html -t lib/raml2html-template.nunjucks -i tmp/raml/wiremock-admin-api.raml -o _docs/wiremock-admin-api.html",
-    "apidocs": "npm run raml2html && shx mkdir -p ../src/main/resources/swagger && api-spec-converter -f raml -t swagger_2 tmp/raml/wiremock-admin-api.raml > ../src/main/resources/swagger/wiremock-admin-api.json"
+    "swaggerui": "shx cp -rf node_modules/swagger-ui-dist/ ../src/main/resources/assets/swagger-ui/",
+    "apidocs": "npm run raml2html && npm run swaggerui && shx mkdir -p ../src/main/resources/swagger && api-spec-converter -c -f raml -t openapi_3 tmp/raml/wiremock-admin-api.raml > ../src/main/resources/swagger/wiremock-admin-api.json"
```

Comment:
Note: I added the `-c` option to the call to `api-spec-converter` to have it check the schema is valid after conversion, which will help prevent typos and whatnot